server/admin/api.go

```diff
@@ -267,6 +267,11 @@ func (s *Server) apiNotifyAll(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "no message to broadcast", http.StatusBadRequest)
 		return
 	}
+	// Remove trailing newline if present. A newline is added by the curl
+	// command when sending from file.
+	if body[len(body)-1] == '\n' {
+		body = body[:len(body)-1]
+	}
```

Comment:
After adding the quotation marks to the notification on the client side, I noticed, and it is in your example too, that there is an extra newline. It looks like curl is adding this when reading from file for some reason. It would of course be better to change the curl command to not add an extra newline, but I'm not sure how to achieve that. Hopefully this addition is harmless.